### ISSUE TITLE
Fire configurationchange event in case the track is changing of source

### ISSUE THREAD

**Issue by youennf** [w3c/mediacapture-screen-share#227]
It seems potentially useful to fire the configurationchange event (defined in mediacapture-extensions) when the source of the track is changing. This may help the web app look at the surface type and other meaningful parameters.

**Comment by eladalon1983** [comment]
Are you thinking [DisplayMediaStreamConstraints.surfaceSwitching](https://github.com/w3c/mediacapture-screen-share/pull/225) here?

**Comment by youennf** [comment]
Right, although the spec already allows to change the track source even without surfaceSwitching.

**Comment by eladalon1983** [comment]
> Right, although the spec already allows to change the track source even without surfaceSwitching.

Yes, surfaceSwitching is just a way for the application to hint to the user agent as to whether it wants such a mechanism to be exposed to the user.

Firing an event of **some** sort seems very useful indeed.
